content/02.body.md

```diff
@@ -0,0 +1,282 @@
+## Background
+
+### Node degree bias
+
+Networks of biomedical data often have node degrees that are biased with respect to the extent to which certain entities have been studied.
+Compared to a hypothetical network of the true relationships, biomedical data networks have edges that are missing in a non-uniform way, where some nodes have a much higher fraction of their true edges while others have almost none of their true edges.
+This particular kind of bias can occur when networks are constructed using data extracted from the literature.
+As the extent to which a biomedical entity has been studied does not necessarily imply its number of true connections, less well-studied entities will have degrees that are biased toward being significantly lower than their true number of connections, when compared to better-studied entities.
+This phenomenon of edges missing in a non-uniform manner is due to "inspection bias", in which certain entities or classes of entities are studied in greater detail than others [@doi:10.1038/nature04209].
+
+Bias in the structure of networks can be demonstrated in domains where unbiased experimentation methods exist.
+While systematic methods are not free of all bias, the bias introduced should be independent of degree.
+If the biases of high-throughput experimentation are roughly degree-independent, then the degree distributions of networks produced via comprehensive, systematic experimentation resemble the degree distributions of true relationships.
+When such unbiased data are available, the degree bias in literature-derived networks can be quantified by comparing degree between literature-derived and unbiased networks.
+
+Figure {@fig:node-degree-bias} gives examples of degree distribution bias in five networks, two protein-protein interaction networks (one of which is literature-derived, and the other of which was constructed via a high-throughput screen of all human proteins), two transcription factor-target gene networks (one of which is literature-derived, and the other of which was constructed from a ChIP-seq screen), and a co-authorship network based on bioinformatics preprints posted on bioRxiv. 
```

Comment:
This gets a bit at the multifunctionality note above. The sentence is pretty long. I think, in this paragraph, you might also want to connect the dots about bioRxiv date cutoff networks being uniformly sampled, while the others are not. The text + figure will get one there, but it's probably better for you to explicitly connect the dots 😁